### PR TITLE
[Tech Debt] Added “Balance” label with localizations

### PR DESCRIPTION
**Pull request by kcw-grunt**
- Set the layout

### CODE REVIEW

File: loafwallet/Storyboards/Main.storyboard [hunk 1]
```diff
@@ -29,11 +29,11 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <tabBar contentMode="redraw" translucent="NO" translatesAutoresizingMaskIntoConstraints="NO" id="j3Z-Gw-zod">
-                                <rect key="frame" x="0.0" y="589" width="414" height="98"/>
+                                <rect key="frame" x="0.0" y="638" width="414" height="49"/>
                                 <viewLayoutGuide key="safeArea" id="TzC-tl-jAG"/>
                                 <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                 <items>
-                                    <tabBarItem tag="0" title="History" image="history_icon" selectedImage="history_icon" id="qwA-9l-CfE" userLabel="History"/>
+                                    <tabBarItem title="History" image="history_icon" selectedImage="history_icon" id="qwA-9l-CfE" userLabel="History"/>
```
Comment (mosadialiou) [inline loafwallet/Storyboards/Main.storyboard:36]: @kcw-grunt Do you really want to remove the `tag="0"` attribute?

File: loafwallet/src/Strings/es.lproj/Localizable.strings [hunk 2]
```diff
@@ -479,6 +479,9 @@
 /* No comment provided by engineer. */
 "Malformed URI" = "URI malformado";
 
+/* Balance */
+"ManageWallet.balance" = "Equilibrio";
```
Comment (mosadialiou) [inline loafwallet/src/Strings/es.lproj/Localizable.strings:483]: Recheck the translation. I think `Saldo` is more appropriated

File: loafwallet/src/Strings/fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -478,6 +478,9 @@
 /* No comment provided by engineer. */
 "Malformed URI" = "URI mal formé";
 
+/* Balance */
+"ManageWallet.balance" = "Équilibre";
```
Comment (mosadialiou) [inline loafwallet/src/Strings/fr.lproj/Localizable.strings:482]: `Solde` is more suited

File: loafwallet/src/Strings/it.lproj/Localizable.strings [hunk 4]
```diff
@@ -478,6 +478,9 @@
 /* No comment provided by engineer. */
 "Malformed URI" = "URI non valido";
 
+/* Balance */
+"ManageWallet.balance" = "Equilibrio";
```
Comment (mosadialiou) [inline loafwallet/src/Strings/it.lproj/Localizable.strings:482]: Recheck please. Put it in a whole phrase to see the correct translation

File: loafwallet/src/Strings/pt.lproj/Localizable.strings [hunk 5]
```diff
@@ -479,6 +479,9 @@
 /* No comment provided by engineer. */
 "Malformed URI" = "URI malformado";
 
+/* Balance */
+"ManageWallet.balance" = "Equilíbrio";
```
Comment (mosadialiou) [inline loafwallet/src/Strings/pt.lproj/Localizable.strings:483]: Please recheck the translation.

**Review by mosadialiou — CHANGES_REQUESTED** [review]

File: loafwallet/Storyboards/Main.storyboard (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet/Storyboards/Main.storyboard:36]: Xcode corrupted this.

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**